File: Beecrowd-Python/1035.py
# ler os quatro valores inteiros A, B, C e D
A, B, C, D = map(int, input(). split())

# verificações
if (B>C) and (D>A) and (C+D>A+B) and (C>0) and (D>0) and (A%2==0):
    print("Valores aceitos")
else: 
    print("Valores nao aceitos")

File: Beecrowd-C/1035.c
#include <stdio.h>
#include <math.h>

int main() {
    // Declaração de variáveis para armazenar os valores A, B, C, D
    int A, B, C, D;

    // Solicita ao usuário que insira os valores A, B, C, D
    scanf("%d%d%d%d", &A, &B, &C, &D);

    // Verifica as condições para aceitar os valores
    if (B > C && D > A && C + D > A + B && C > 0 && D > 0 && A % 2 == 0) {
        // Se as condições são atendidas, imprime "Valores aceitos"
        printf("Valores aceitos\n");
    } else {
        // Se as condições não são atendidas, imprime "Valores nao aceitos"
        printf("Valores nao aceitos\n");
    }

    return 0;
}